app.js

```diff
@@ -13,6 +14,21 @@ if (config && config.http && config.http.port)
 else
   port = 3000;
 
+// copy client-side script from dependency
+function copy(fromPath, toPath) {
+  var fromStream = fs.createReadStream(path.join(__dirname, fromPath));
+  var toStream = fs.createWriteStream(path.join(__dirname, toPath));
+  fromStream.pipe(toStream);
+}
+copy('node_modules/nodep-date-input-polyfill/nodep-date-input-polyfill.dist.js',
+     'public/scripts/nodep-date-input-polyfill.dist.js');
+copy('node_modules/jquery/dist/jquery.min.js',
+     'public/scripts/jquery.min.js');
+copy('node_modules/jquery/dist/jquery.min.map',
+     'public/scripts/jquery.min.map');
+copy('node_modules/jquery/dist/jquery.js',
+     'public/scripts/jquery.js');
```

Comment:
Not asking you to do this before merge (this applies to any comments I leave here), but I'd prefer to just hardcode these files into the repo than run system disk access calls on app startup.